src/malloc_wrapper.cc

```diff
@@ -46,7 +46,7 @@ void gnu_backtrace()
 	}
 
 	for (j = 0; j < nptrs; j++)
```

Comment:
[llvmlibc-implementation-in-namespace]: declaration must be declared within the '__llvm_libc' namespace